### ISSUE TITLE
Releases API gives errors

### ISSUE THREAD

**Issue by loewis** [python/pythondotorg#496]
When I try to access

https://www.python.org/api/v1/downloads/release/?format=json&version=3

it says

```
{"error": "The model '<Release: Python 3.1.5>' has an empty attribute 'release_page' and doesn't allow a null value."}
```

Likewise, for 

https://www.python.org/api/v1/downloads/release/?format=json&limit=100

I get

```
{"error": "The model '<Release: Python 2.7.8>' has an empty attribute 'release_page' and doesn't allow a null value."}
```

**Comment by birkenfeld** [comment]
Related to #481.

(Interestingly, the admin interface allows clearing the "Release page" field, which is as it should be - for new releases we don't want separate pages anymore.)

**Comment by benjaminp** [comment]
I fixed this a few weeks ago in 36b6e49cfd5a8302e9240e77062597a0e3d2cab7.
